gateway/fedimint-lightning/src/ldk.rs

```diff
@@ -473,19 +480,23 @@ impl ILnRpcClient for GatewayLdkClient {
         SendOnchainRequest {
             address,
             amount,
-            // TODO: Respect this fee rate once `ldk-node` supports setting a custom fee rate.
-            // This work is planned to be in `ldk-node` v0.4 and is tracked here:
-            // https://github.com/lightningdevkit/ldk-node/issues/176
-            fee_rate_sats_per_vbyte: _,
+            fee_rate_sats_per_vbyte,
         }: SendOnchainRequest,
     ) -> Result<SendOnchainResponse, LightningRpcError> {
         let onchain = self.node.onchain_payment();
 
+        let retain_reserves = false;
```

Comment:
Should we create an issue to change to `true` once the upstream bug is fixed?

Reply:
Do we need another one in our repo? I might just track using this https://github.com/lightningdevkit/ldk-node/issues/539

Reply:
Oh nice, thanks for opening that. Just a single issue upstream is great :+1: